### PR TITLE
chore(functions/v2/log): Update log samples to cloud event declarative signature

### PR DESCRIPTION
**Pull request by kenneth-rosario**

### CODE REVIEW

File: functions/v2/log/main.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    """
+    print("Hello, stdout!")
+
+
+# [END functions_cloudevent_log_helloworld]
+
+
+# [START functions_cloudevent_log_stackdriver]
+@functions_framework.cloud_event
+def process_log_entry(event):
```
Comment (engelke) [inline functions/v2/log/main.py:39]: I'm unclear on the context here. A Cloud Function has only one entry point specified, so why does this code seem to have two? Would it be deployed twice, once with each entry point? Should the two entry points be two separate samples?

**Review by engelke — CHANGES_REQUESTED** [review]
Please clarify the use of two entry points in this sample.

File: functions/v2/log/main.py (hunk 1 above)
Comment (kenneth-rosario) [reply functions/v2/log/main.py:39]: I was updating https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/functions/log/main.py that had both samples together. I will separate them into different samples which is cleaner.

Comment (kenneth-rosario) [reply functions/v2/log/main.py]: I split the sample into two distinct ones in their respective directories.

**Review by engelke — APPROVED** [review]
LGTM

**Pull request merged**